### PR TITLE
James

### PR DESCRIPTION
**Pull request by Jlpowellv**
added addItem and editItem functionality

### CODE REVIEW

File: src/com/main/db/Database.java [hunk 1]
```diff
@@ -30,7 +36,119 @@ public void dbClose() {
...
+	 * 
+	 * @parameters
+	 * name --> is the name of an item that the vendor has 
+	 * vendorId --> is the vendor identification number
+	 * 
+	 * the purpose of this function is to validate that the is an item with name "name" that belongs 
+	 * to vendorId
+	 * 
+	 * 
+	 * ********this function is no longer used as of the creation of fetchItems********
```
Comment (Kev-Index) [inline src/com/main/db/Database.java:73]: Do we still need this function if its not being used anymore? Maybe comment the function out

Comment (Jlpowellv) [reply src/com/main/db/Database.java:73]: there is other stuff I can change that is small too, I will do it tomorrow

**Review by Kev-Index — APPROVED** [review]

File: src/com/main/db/Database.java [hunk 2]
```diff
@@ -30,11 +36,123 @@ public void dbClose() {
...
+		dbConnect();
+		String stmt = "insert into item(vendorId,name,price,quantity)values (?,?,?,?);";
+		try {
+			PreparedStatement p = con.prepareStatement(stmt);
+			p.setInt(1, i.getVendorId());
+			p.setString(2, i.getName());
+			p.setFloat(3, i.getPrice());
+			p.setInt(4, i.getQuantity());
+			p.executeUpdate();
+			System.out.println("Item successfully added to your menu!!!");
```
Comment (Kev-Index) [inline src/com/main/db/Database.java:55]: I like the enthusiasm

**Review by sshJerry — APPROVED** [review]
Looks good

**Pull request merged**